### DISCUSSION TITLE
Separate dependencyDashboards based on packageRules

### DISCUSSION THREAD

**Discussion by mjnagel** [renovatebot/renovate#13989] [Request Help]
Our team has a repo where we want to monitor a number of dependencies. These dependencies naturally fall into two overlapping groups - all are docker dependencies, but we have 4 docker dependencies from the same docker repository. Right now we have packageRules that look roughly like this:
```
    "packageRules": [
        {
            "matchDatasources": ["docker"],
            "groupName": "Docker",
        },
        {
            "matchPackagePatterns": ["<name>"],  // name is the repository name so it matches all 4 dependencies
            "additionalBranchPrefix": "<name>-",
            "groupName": "<name>"
        }
    ],
```
This gets us separate MRs for each of these two groups and that works perfectly, but we would also like the MRs to be linked to separate dependencyDashboards (issues) so that we can:
- have different titles for each (i.e. "Update General Docker Dependencies" and "Update <name> Dependencies"
- have different descriptions for each (basically we add checkboxes for things to validate when taking on one of the issues and getting the MR ready)
- allow different team members to assign issues to different sprints and work them independently without confusion.

I tried a number of things with packageRules but was unable to find anything that does what I want. I'm not sure if this just isn't supported or if I'm just not seeing the config options for it? Any help is appreciated...our other option is disabling the dependencyDashboard and adding automation on our end that opens an issue per MR that the renovate user opens, but would certainly prefer to do this natively if possible.

**Comment by viceice** [comment]
Only a single dashboard is supported

**Reply by mjnagel** [reply]
Is this anything you'd envision supporting in the future?
